common/src/ui/RecentDocumentListBox.cpp

```diff
@@ -25,6 +25,8 @@
 #include "io/ResourceUtils.h"
 
 #include <cassert>
+#include <QKeyEvent>
```

Comment:
TB uses clang-format to automatically format the code and sort includes, you need to set that up and run it on the code before committing. See https://github.com/TrenchBroom/dev-tools for binaries for your platform.